openedx/adg/lms/registration_extension/forms.py

```diff
@@ -158,3 +161,24 @@ class ExtendedProfileForm(ModelForm):
     class Meta(object):
         model = ExtendedUserProfile
         fields = ('company',)
+
+
+def get_registration_form_override(request, form_desc):
+    """
+    It will add default values in the form if user is invited
+    """
+    email = request.GET.get("email", None)
+    email = parse.unquote(email)
+    invitation_obj = Invitation.objects.filter(email=email).first()
+    if invitation_obj:
+        business_unit = invitation_obj.business_unit
+        for form in form_desc.fields:
+            if form["name"] == "email":
+                form["defaultValue"] = email
+                form["restrictions"]["readonly"] = True
+            if form["name"] == "is_adg_employee":
+                form["defaultValue"] = bool(business_unit)
+            if business_unit and form["name"] == "company":
```

Comment:
We should either use `elif` or `continue` once a condition is true else it will check all conditions.